tensorboard/plugins/base_plugin.py

```diff
@@ -196,3 +196,10 @@ def __init__(self, plugin_class):
 
   def load(self, context):
     return self._plugin_class(context)
+
+
+class FlagsException(ValueError):
```

Comment:
Please modify the docs for `TBLoader.fix_flags` to indicate that
implementors may raise this exception to signal an error.

Reply:
Added a "Raises: " section to the doc.